starboard/loader_app/record_loader_app_status.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef STARBOARD_LOADER_APP_RECORD_LOADER_APP_STATUS_H_
+#define STARBOARD_LOADER_APP_RECORD_LOADER_APP_STATUS_H_
+
+#include <string>
+
+#include "starboard/extension/loader_app_metrics.h"
+
+namespace starboard {
+namespace loader_app {
+
+// Record the slot selection status
+void RecordSlotSelectionStatus(SlotSelectionStatus status);
```

Comment:
Is there another verb you would like in this name instead of "record"?

I think "record" works well here in isolation, but we already use it in `cobalt::browser::RecordLoaderAppMetrics`, where "record" means to call the upstream histogram APIs to register the histograms with the telemetry system (and I borrowed "record" from the function comments for these APIs).

I just don't want the name of this function to imply that it's interacting with the upstream histogram APIs.

Reply:
Reopening this as I don't see a response or any changes. Please lmk what you think.

Reply:
IMO it's okay to use "record". We also used "RecordStatus" in third_party/crashpad/wrapper/wrapper.cc

Reply:
Sure. I think at least a more descriptive function comment could help then - describing that this function persists the status so that it can be read in the Cobalt layer - but up to you.